cronjob.js

```diff
@@ -94,6 +95,14 @@ const initCronJob = () => {
       postSlackMessage(message);
     }
   });
+
+  const jobs = schedule.scheduleJob('0 9-21/2 * * 1-6', async () => {
```

Comment:
assign this cron timing to a variable and add  a comment for the developers to know what is the exact time of run